tailwind.config.js

```diff
@@ -19,6 +19,33 @@ module.exports = {
         serif: ["var(--font-sora)", ...fontFamily.serif],
         scribble: ["var(--font-scribble)", ...fontFamily.mono],
       },
+      keyframes: {
+        wave: {
+          "0%": { transform: "rotate(0.0deg)" },
+          "10%": { transform: "rotate(14deg)" },
+          "20%": { transform: "rotate(-8deg)" },
+          "30%": { transform: "rotate(14deg)" },
+          "40%": { transform: "rotate(-4deg)" },
+          "50%": { transform: "rotate(10.0deg)" },
+          "60%": { transform: "rotate(0.0deg)" },
+          "100%": { transform: "rotate(0.0deg)" },
+        },
+        grow_and_shrink: {
+          "0%": { transform: "scale(1)" },
+          "50%": { transform: "scale(1.5)" },
+          "100%": { transform: "scale(1)" },
+        },
+        slide_in: {
+          from: { transform: "translateX(100%)" },
+          to: { transform: "translateX(0%)" },
+        },
```

Comment:
Can you use _snake case_ 🐍 please 🙏🏻 ? You can wrap it in _quotes_ so it works 😉 

```js
"slide-in": {
  from: { transform: "translateX(100%)" },
  to: { transform: "translateX(0%)" },
},
```

Reply:
Done e059781b42d2759f64b649d236e2930810599979